packages/hawtio/src/core/core.ts

```diff
@@ -13,6 +13,9 @@ export interface Plugin {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   component: React.ComponentType<any>
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  toolbarItems?: React.ComponentType<any>[]
```

Comment:
I would name it `headerItems` instead of `toolbarItems`, as there can be other toolbars in one page such as table toolbars.